teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorOptions.java

```diff
@@ -130,6 +130,16 @@ public class ValidatorOptions {
       fallbackValue = "true")
   private boolean blockV3Enabled = ValidatorConfig.DEFAULT_BLOCK_V3_ENABLED;
 
+  @Option(
+      names = {"--allow-no-loaded-keys"},
+      paramLabel = "<BOOLEAN>",
+      description = "Enable allowing no loaded validator keys",
```

Comment:
maybe something like `exit-when-no-validator-keys-enabled`

would at least need to finish with `enabled`

it would not be hidden...

The description would be something like "This option will terminate the process If no validator keys are found during startup"